.storybook/webpack.config.js

```diff
@@ -1,10 +1,17 @@
 const path = require('path');
-const { DefinePlugin } = require('webpack');
+const { DefinePlugin, IgnorePlugin } = require('webpack');
 
-if (
-  process.env.STORYBOOK_THEME_LOCATION === 'external' &&
-  !process.env.STORYBOOK_BRANDING_PATH
-) {
+const {
+  STORYBOOK_THEME_LOCATION,
+  STORYBOOK_BRANDING_PATH,
```

Comment:
do these need to be added to the sample env file?

Reply:
Not the ThemeLocation one, thats being set from the build scripts.. The other is in the sample already.